safe/utilities/direction_distance.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
+
+"""Direction and Distance Functionality."""
+
+import logging
+from qgis.core import QgsDistanceArea
+
+LOGGER = logging.getLogger('InaSAFE')
+
+
+def bearing_to_cardinal(angle):
+    """Get cardinality of an angle.
+
+    :param angle: Bearing angle
+    :return:
```

Comment:
Give return value, and the type. Also type for the angle. Is it in radian, degree, or gradient ?

Reply:
Sure, I will provide more complete information in the docstring